grove/connectors/oomnitza/api.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+"""Oomnitza API client.
+
+As Oomnitza SDK does not currently support the Activities API, this client has been created in
+the interim.
+"""
+
+import logging
+from typing import Dict, Optional
+
+import requests
+from grove.exceptions import RequestFailedException
+from grove.types import AuditLogEntries, HTTPResponse
+
+API_BASE_URI = "https://{identity}.oomnitza.com"
+API_PAGE_SIZE = 200
+
+
+class Client:
+    def __init__(
+        self,
+        identity: Optional[str] = None,
+        token: Optional[str] = None,
+    ):
+        """Setup a new client.
+
+        :param identity: The name of the Oomnitza organisation.
+        :param token: The Oomnitza API token.
+        """
+        self.logger = logging.getLogger(__name__)
+        self.headers = {
+            "content-type": "application/json",
+            "Authorization2": f"{token}",
```

Comment:
No action, but reference for future reviewers: I thought this might be a typo, but it looks like this is correct based on some [OSS Oomnitza code](https://github.com/Oomnitza/oomnitza-connector/blob/cdfabbc5eb9995e15731c59cd2ffc0a682b06eb2/connectors/oomnitza.py#L40).